src/styles/blocks/header.scss

```diff
@@ -0,0 +1,13 @@
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
```

Comment:
Consider using relative units (like percentages or ems) for padding to improve responsiveness, especially for the `padding` property.